tests/test_requirements.txt

```diff
@@ -0,0 +1,2 @@
+nox==2025.2.9
```

Comment:
Here the file!, Do you think is good idea to add pyenv?

Reply:
Sure!

Reply:
Thanks, Done: [2542ec0](https://github.com/ivica-k/terraform-validate-explorer/pull/13/commits/2542ec0e8c213da233b608d1591144590a9695a5)